Elementary-Celebrity-Recognition-My-Dear-Watson/SwiftCloudWorkshop2.swift

```diff
@@ -53,10 +55,10 @@ func main(args: [String:Any]) -> [String:Any] {
     // Watson Visual Recognition API call.
     let semaphore = DispatchSemaphore(value: 0)
     
-    // This is the closure that the Watson Visual Recognition API will used in the case of a failure.
+    // This is the closure that gets called if the Watson Visual Recognition API returns an error.
```

Comment:
Good revision. Also, I now notice that there is a grammatical mistake in my original comment. Ah, late night coding. 😆